addons/account_tax_python/account_tax_python.py

```diff
@@ -25,7 +27,7 @@ class AccountTaxPython(models.Model):
     def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, partner=None):
         self.ensure_one()
         if self.amount_type == 'code':
-            localdict = {'base_amount': base_amount, 'price_unit':price_unit, 'quantity': quantity, 'product':product, 'partner':partner}
+            localdict = {'base_amount': base_amount, 'price_unit':price_unit, 'quantity': quantity, 'product':product, 'partner':partner, 'company': company}
```

Comment:
Missing company variable assignment, this will probably crash. Also I don't understand why you need to add company to the localdict of the exec. Is it needed for this pull request?